cypress/integration/dev.spec.js

```diff
@@ -0,0 +1,40 @@
+/* global describe, it, cy */
+
+describe('Dev Page', () => {
+  it('loads', () => {
+    cy.visit('/dev');
+    cy.contains('Hands Down!');
+  });
+
+  it('has the important elements', () => {
+    cy.get('#output');
+    cy.contains('0 Data Points');
+  });
+
+  it('can turn on collection', () => {
+    const collectionButton = cy.get('#collection-state-btn');
+    collectionButton.contains('Off');
+    /* collectionButton.click().then((btn) => {
```

Comment:
why's this (and testing other buttons' functionalities) commented out?

Reply:
Because of some inconsistencies I couldn't wrap my head around until now.